ui/search/nuxeo-results-view.js

```diff
@@ -0,0 +1,453 @@
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+import { html } from '@polymer/polymer/lib/utils/html-tag.js';
+import { mixinBehaviors } from '@polymer/polymer/lib/legacy/class';
+import '@polymer/iron-flex-layout/iron-flex-layout.js';
+import '@polymer/iron-collapse/iron-collapse.js';
+import '@nuxeo/nuxeo-elements/nuxeo-page-provider.js';
+import { I18nBehavior } from '../nuxeo-i18n-behavior.js';
+import './nuxeo-search-form-layout.js';
+import './nuxeo-search-results-layout.js';
+
+{
+  /**
+   * `nuxeo-results-view`
+   * @group Nuxeo UI
+   * @appliesMixin Nuxeo.I18nBehavior
+   * @element nuxeo-results-view
+   */
+  class ResultsView extends mixinBehaviors([I18nBehavior], Nuxeo.Element) {
+    static get template() {
+      return html`
+        <style include="nuxeo-styles iron-flex iron-flex-alignment">
+          .ellipsis {
+            text-overflow: ellipsis;
+            overflow: hidden;
+            white-space: nowrap;
+            display: block;
+          }
+
+          .capitalize {
+            text-transform: capitalize;
+          }
+
+          .form {
+            @apply --paper-card;
+            padding: 0;
+          }
+
+          .header,
+          #collapse {
+            padding: 0 16px;
+          }
+
+          .buttons {
+            @apply --buttons-bar;
+          }
+
+          .hidden {
+            visibility: hidden;
+          }
+
+          .count {
+            @apply --layout-horizontal;
+            @apply --layout-center;
+            @apply --layout-center-justified;
+            background: var(--nuxeo-badge-background);
+            color: white;
+            border-radius: 50%;
+            width: 24px;
+            height: 24px;
+            margin-left: 1em;
+          }
+
+          .buttons {
+            @apply --layout-horizontal;
+          }
+
+          paper-spinner-lite {
+            width: 22px;
+            height: 22px;
+            padding-left: 8px;
+          }
+        </style>
+
+        <nuxeo-page-provider
+          id="provider"
+          provider="[[provider]]"
+          page-size="[[pageSize]]"
+          aggregations="{{aggregations}}"
+          enrichers="[[enrichers]]"
+          params="[[_params]]"
+          quick-filters="{{quickFilters}}"
+          schemas="[[schemas]]"
+          loading="{{loading}}"
+          headers="[[headers]]"
+          fetch-aggregates
+          skip-aggregates$="[[skipAggregates]]"
+          on-error="_onError"
+        >
+        </nuxeo-page-provider>
+
+        <template is="dom-if" if="[[showFilters]]">
+          <div class="form">
+            <div class="header horizontal layout center">
+              <a href="javascript:undefined" class="horizontal layout center" on-tap="toggleExpand">
```

Comment:
Since we are not navigating to anywhere, it seems to me the best element to be used here would be a button, right?
```suggestion
              <button class="horizontal layout center" on-tap="toggleExpand" aria-label$="[[i18n('command.expand')]]">
```
To accomplish this we would also need to reset the native style button. To accomplish this we could take inspiration on [what we did here](https://github.com/nuxeo/nuxeo-elements/blob/master/ui/nuxeo-aggregation/nuxeo-checkbox-aggregation.js#L84-L106)

Reply:
On a second thought, maybe this aria-label isn't accurately describing what we are doing with this button. We can consider adding instead using an `aria-labelledby` to the button and `id` to the last span child.